src/views/dashboard/related/trace/components/D3Graph/Index.vue

```diff
@@ -60,6 +61,9 @@ limitations under the License. -->
     }
     draw();
     loading.value = false;
+    emitter.on("trace_resize", () => {
```

Comment:
You should use the [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/MutationObserver) to observe `div` change, and use `debounceFunc` to draw trace graph.

Reply:
Please remove the `emitter`.

Reply:
Thank you so much, this is a very convenient way to do dom monitoring, learned.